src/game/boe.main.cpp

```diff
@@ -361,32 +355,36 @@ void handle_events() {
 	cFramerateLimiter fps_limiter;
 
 	while(!All_Done) {
+		if(replaying && has_next_action()){
+			replay_next_action();
+		}else{
```

Comment:
I guess it's not super important, but you could've avoided indenting all this code by just adding `continue` on the line above…